Module 5/Problem 10/problem10.json

```diff
@@ -0,0 +1,67 @@
+Input:
+
+{
+  "name": "John Smith",
+  "age": 35,
+  "address": {
+    "street": "123 Main St",
+    "city": "Anytown",
+    "state": "CA",
+    "zip": "12345"
+  },
+  "phone_numbers": [
+    {
+      "type": "home",
+      "number": "555-1234"
+    },
+    {
+      "type": "work",
+      "number": "555-5678"
+    }
+  ]
+}
+
+Transformation:
+
+[
+  {
+    "operation": "modify-overwrite-beta",
+    "spec": {
+      "fullname": "=split(' ',@(2,name))",
+      "first_name": "=firstElement(@(2,fullname))",
+      "last_name": "=lastElement(@(2,fullname))",
```

Comment:
It is giving the desired output but 2 is not the correct level here. Please check.